python/ray/services.py

```diff
@@ -1345,6 +1323,26 @@ def start_raylet(redis_address,
 
     gcs_ip_address, gcs_port = redis_address.split(":")
 
+    has_java_command = False
+    try:
+        java_proc = subprocess.run(
```

Comment:
Will this command output be printed to console?

Reply:
No. `subprocess.PIPE` means capturing the output.